loadtest/store/store.go

```diff
@@ -7,6 +7,16 @@ import (
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
+// SelectionType is the selection parameter for a store entity
+type SelectionType uint8
+
+// Defines the membership rules for a store entity
+const (
+	SelectMemberOf SelectionType = iota + 1
```

Comment:
It should not be "iota +1". That doesn't shift the bit, it adds one to each const. I was suggesting "1 << iota". That will keep shifting the bit. See: https://golang.org/pkg/os/#FileMode and https://golang.org/pkg/net/#Flags for example.

Something like:

```go
type SelectionType uint8

const (
 SelectMemberOf SelectionType = 1 << iota // Select all cases where the user is a member
 SelectNotMemberOf // Select cases where the user is not a member

SelectAny = SelectMemberOf |  SelectNotMemberOf
)
```

This means memberOf is 1 << 1 = 1, notmember is 1 << 2 = 2 and selectAny is therefore 1|2=3. Your code did the same thing, but arrived with a different logic.

The whole idea is that each bit indicates the presence of something. And so we can do an OR operation to signify the presence of multiple bits. This lets us do all sorts of interesting things using just 8 bits. Hope that clarified my point. :)

Reply:
I see, somehow I thought we won't need `SelectAny` at all and use `SelectMemberOf |  SelectNotMemberOf` when its needed :) Will change it.